src/Expectation.php

```diff
@@ -339,4 +340,9 @@ public static function hasMethod(string $name): bool
             || method_exists(Mixins\Expectation::class, $name)
             || self::hasExtend($name);
     }
+
+    public function any(): Any
```

Comment:
Can you add a comment explain what this does? You may want to check Jest here.

Reply:
Do you think that "Matches any argument" would be enough as explanation?

Reply:
done